### PR TITLE
Add djsonb to setup.py

### PR DESCRIPTION
**Pull request by moradology**
Integrate ashlar with our fork of django-pgjson

### CODE REVIEW

File: ashlar/filters.py [hunk 1]
```diff
@@ -162,35 +163,26 @@ class JsonBFilterBackend(BaseFilterBackend):
     """
     def filter_queryset(self, request, queryset, view):
         """ Filter by configured jsonb_filters on jsonb_filter_field """
+        lookup_name = 'jsonb'
         filter_field = getattr(view, 'jsonb_filter_field', None)
```
Comment (ddohler) [inline ashlar/filters.py:158]: Not part of this PR, but does this mean that it's not possible to use this filter on models that have more than one JSONB field?

Comment (moradology) [inline ashlar/filters.py:158]: Currently that's how I set it up - I figured we could add this logic back as necessary just so that this filter could be cleaner to reason about now

File: ashlar/migrations/0001_initial.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ class Migration(migrations.Migration):
                 ('created', models.DateTimeField(auto_now_add=True)),
                 ('modified', models.DateTimeField(auto_now=True)),
                 ('version', models.IntegerField()),
-                ('schema', django_pgjson.fields.JsonBField()),
+                ('schema', djsonb.fields.JsonBField()),
```
Comment (ddohler) [inline ashlar/migrations/0001_initial.py]: Is it feasible to create a new migration for this rather than editing old migrations? That has caused problems in the past because it requires a database drop / reprovision.

File: ashlar/filters.py (hunk 1 above)
Comment (ddohler) [inline ashlar/filters.py:158]: That makes sense; it'd be good to add a note mentioning the restriction.

**Comment by ddohler** [comment]
+1
